autovacuum_message_attachment/views/rule_vacuum.xml

```diff
@@ -20,39 +20,30 @@
                             <field name="company_id" />
                             <field name="retention_time" />
                         </group>
-                        <group
-                            col="4"
-                            attrs="{'invisible': [('ttype', '!=', 'message')]}"
-                        >
-                            <field
-                                name="message_type"
-                                attrs="{'required': [('ttype', '=', 'message')]}"
-                            />
+                        <group col="4" invisible="ttype != 'message'">
+                            <field name="message_type" required="ttype == 'message'" />
                             <field name="empty_subtype" />
                         </group>
-                        <group
-                            string="Message Subtypes"
-                            attrs="{'invisible': [('ttype', '!=', 'message')]}"
-                        >
+                        <group string="Message Subtypes" invisible="ttype != 'message'">
                             <field name="message_subtype_ids" nolabel="1" colspan="4" />
                         </group>
-                        <group attrs="{'invisible': [('ttype', '!=', 'attachment')]}">
+                        <group invisible="ttype != 'attachment'">
                             <field name="filename_pattern" />
                             <field name="inheriting_model" />
                             <field
                                 name="empty_model"
-                                attrs="{'invisible': ['|', ('model_ids', '!=', []), ('filename_pattern', '=', False)]}"
+                                invisible="model_ids != [] or not filename_pattern"
                             />
                         </group>
                         <group string="Models">
                             <field name="model_ids" nolabel="1" colspan="4" />
                         </group>
                         <group>
-                            <field name="model_id" />
+                            <field name="model_id" readonly="1" />
```

Comment:
You can remove `readonly="1"` as both `model` and `model_id` are compute fields.